src/OpenLoco/src/Windows/IndustryList.cpp

```diff
@@ -275,6 +283,50 @@ namespace OpenLoco::Ui::Windows::IndustryList
             return rhsVar < lhsVar;
         }
 
+        static uint32_t getProductionLastMonth(const OpenLoco::Industry& industry, StringId* unitType = nullptr)
+        {
+            auto industryObj = ObjectManager::get<IndustryObject>(industry.objectId);
+            uint32_t cargoProduction = std::numeric_limits<uint32_t>::max();
+
+            if (unitType)
+            {
+                *unitType = StringIds::empty;
+            }
+
+            if (industryObj->producesCargo())
+            {
+                auto cargoNumber = 0;
+                for (const auto& producedCargoType : industryObj->producedCargoType)
+                {
+                    if (producedCargoType != IndustryObject::cargoTypeNull)
+                    {
+                        cargoProduction = industry.producedCargoQuantityPreviousMonth[cargoNumber];
+
+                        // If the callee is interested in the unit, return it
+                        if (unitType)
+                        {
+                            auto cargoObj = ObjectManager::get<CargoObject>(producedCargoType);
+                            *unitType = cargoProduction > 1 ? cargoObj->unitNamePlural : cargoObj->unitNameSingular;
+                        }
+                        break; // Only support one cargo type for now - it seems most (all?) industries only have a single cargo type
+                    }
+
+                    cargoNumber++;
+                }
+            }
+
+            return cargoProduction;
+        }
+
+        static bool orderByProductionLastMonth(const OpenLoco::Industry& lhs, const OpenLoco::Industry& rhs)
+        {
+            auto lhsVar = getProductionLastMonth(lhs);
```

Comment:
you can probably just `return getProductionLastMonth(rhs) < getProductionLastMonth(lhs);`, i don't think the temporary variables add anything

Reply:
I'm just following the code style from elsewhere in the file. I agree it can be a one-liner.